Yosemite/YosemiteTests/Stores/InboxNotesStoreTests.swift

```diff
@@ -157,13 +157,13 @@ final class InboxNotesStoreTests: XCTestCase {
         XCTAssertFalse(result.isSuccess)
     }
 
-    func test_dismissInboxNote_then_it_update_inbox_note_upon_successful_response() {
+    func test_dismissInboxNote_do_nothing_if_no_inbox_notes_exist_upon_successful_response() {
         // Given a stubbed inbox note network response
         let sampleInboxNoteID: Int64 = 296
-        network.simulateResponse(requestUrlSuffix: "admin/notes/\(sampleInboxNoteID)", filename: "inbox-note")
+        network.simulateResponse(requestUrlSuffix: "admin/notes/delete/\(sampleInboxNoteID)", filename: "inbox-note")
```

Comment:
nit: maybe an assert on storage inbox notes would be useful for comparing this and the next test case:
```suggestion
        network.simulateResponse(requestUrlSuffix: "admin/notes/delete/\(sampleInboxNoteID)", filename: "inbox-note")
        XCTAssertEqual(storedInboxNotesCount, 0)
```